### PR TITLE
Default Streamlit App

### PR DESCRIPTION
**Pull request by Preetesh21**
PR for the issue #60 
Removed all the files and added the required files.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import streamlit as st
+from PIL import Image
+import numpy as np
+import time
+
+
+st.beta_set_page_config(
+page_title="Title of the webpage",
```
Comment (smaranjitghose) [inline app.py:8]: Set Title to AutoVaidya

File: app.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# Just making sure we are not bothered by File Encoding warnings
+st.set_option('deprecation.showfileUploaderEncoding', False)
+
+def main():
+    menu = ['Home', 'Project1','Project2','Project3']
+    choice = st.sidebar.selectbox("Menu", menu)
+
+    if choice == "Home":
+        # Let's set the title of our awesome web app
+        st.title('Title of your Awesome App')
```
Comment (smaranjitghose) [inline app.py:22]: Autovaidya

File: app.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        # Now setting up a header text
+        st.subheader("By Your Cool Dev Name")
```
Comment (smaranjitghose) [inline app.py:24]: Not  Required

File: app.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        # adding random text
+        st.text("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.")
```
Comment (smaranjitghose) [inline app.py:26]: Put a quote related to health here

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
@Preetesh21 whenever you give PRs for the work related to the streamlit app, please try to attach screenshots of the demo of the app in your local system. A gif is preferred more

File: app.py (hunk 1 above)
Comment (Preetesh21) [reply app.py]: Done sir

File: app.py (hunk 2 above)
Comment (Preetesh21) [reply app.py]: Done sir

File: app.py (hunk 3 above)
Comment (Preetesh21) [reply app.py]: Removed sir

File: app.py (hunk 4 above)
Comment (Preetesh21) [reply app.py]: Done sir

**Comment by Preetesh21** [comment]
Done sir and the attached gif
ezgif com-gif-maker

**Comment by smaranjitghose** [comment]
Awesome!
One small suggestion for contribution to open source projects. Whenever you get feedback on PRs and do it, try to give a squash commit by rebasing [ This might sound weird at first but if you Google about it and spend 10-15 min, in the future it will really be helpful for you]

**Pull request merged**